ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -276,3 +276,21 @@ func ForceNewDeploymentFlag() []cli.Flag {
 		},
 	}
 }
+
+func resourceTagsFlag() []cli.Flag {
```

Comment:
Any reason not to combine `resourceTagsFlag` and `disableECSManagedTagsFlag` as one method that contains both flags? I assume any command that takes one flag would take the other; that would make the Flags field under each command a tad less verbose.